js/src/components/experience-rating-banner/feedback-modal.js

```diff
@@ -18,7 +20,22 @@ import AppButton from '../app-button';
  * @param {Function} props.onRequestClose - Function to call when the modal is closed.
  * @return {JSX.Element} The FeedbackModal component.
  */
+
+/**
```

Comment:
@mukeshpanchal27 Can you please add a description as when this event is fired please? Same for the event below.